src/tests/mocks/handlers/clinicalTrialsGov.handler.ts

```diff
@@ -0,0 +1,11 @@
+import { rest } from 'msw';
+import * as process from 'node:process';
+import { responseJSON } from './information/clinicalTrialsGovInfo';
+
+const BASE_URL = 'https://clinicaltrials.gov';
+
+export const clinicalTrialsGovHandlers = [
+  rest.get(`${BASE_URL}/api/v2/studies?pageSize=2`, (request, response, context) => {
```

Comment:
extract the page size as a param, you want to mock every call right? not only if the page size is 2